airbyte-webapp/src/hooks/services/Experiment/ExperimentService.test.tsx

```diff
@@ -4,17 +4,33 @@ import React from "react";
 import { EMPTY, Subject } from "rxjs";
 
 import { Experiments } from "./experiments";
-import { ExperimentProvider, useExperiment } from "./ExperimentService";
+import { ExperimentProvider, ExperimentService, useExperiment } from "./ExperimentService";
+
+type TestExperimentValueType = Experiments["connector.orderOverwrite"];
+
+const TEST_EXPERIMENT_KEY = "connector.orderOverwrite";
+
+const getExperiment: ExperimentService["getExperiment"] = (key) => {
+  if (key === TEST_EXPERIMENT_KEY) {
+    return { test: 13 };
+  }
+  throw new Error(`${key} not mocked for testing`);
+};
```

Comment:
@edmundito I pushed a small change to this, so the method is throwing for not implemented ones, which saves us a couple of `any` and ensures we'll be able to catch it during a test if it would ever be called with a different key.